contracts/factory/src/contract.rs

```diff
@@ -575,6 +577,26 @@ pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, Co
 
                 CONFIG.save(deps.storage, &new_config)?;
```

Comment:
Here the contract will be migrated to the latest version as well, which is 1.2.0 but is_generator_disabled will not be added to PAIR_CONFIGS. We need to add it too

Reply:
Done, thanks